src/lib.rs

```diff
@@ -1935,12 +1943,30 @@ mod tests {
     const U32_BITS: usize = 32;
 
     #[test]
-    fn test_to_str() {
-        let zerolen = BitVec::new();
-        assert_eq!(format!("{:?}", zerolen), "");
+    fn test_display_output() {
+        assert_eq!(format!("{}", BitVec::new()), "");
+        assert_eq!(format!("{}", BitVec::from_elem(1, true)), "1");
+        assert_eq!(format!("{}", BitVec::from_elem(8, false)), "00000000")
+    }
 
-        let eightbits = BitVec::from_elem(8, false);
-        assert_eq!(format!("{:?}", eightbits), "00000000")
+    #[test]
+    fn test_debug_output() {
+        assert_eq!(
+            format!("{:?}", BitVec::new()),
+            "BitVec { storage: \"\", nbits: 0 }"
+        );
+        assert_eq!(
+            format!("{:?}", BitVec::from_elem(1, true)),
+            "BitVec { storage: \"1\", nbits: 1 }"
+        );
+        assert_eq!(
+            format!("{:?}", BitVec::from_elem(8, false)),
+            "BitVec { storage: \"00000000\", nbits: 8 }"
+        );
+        assert_eq!(
+            format!("{:?}", BitVec::from_elem(33, true)),
+            "BitVec { storage: \"11111111111111111111111111111111 1\", nbits: 33 }"
```

Comment:
Could add a test for assymetric like `storage: \"100000110101\"` with `BitVec::from_bytes()`